feature-configs/5g-ref-nw/profile-base/ran-general/kustomization.yaml

```diff
@@ -1,2 +1,3 @@
 resources:
+- ../../../deploy/container-mount-namespace
```

Comment:
Adding this to profile-base/ran-general will add this to all DU profiles, not just SNO as the commit text implies, won't it?

For the record, I don't think that's a wrong thing to do, but I think the commit message should be more clear that this is adding this feature to all RAN profiles, not just SNO.